Makefile

```diff
@@ -166,6 +166,7 @@ mypy:
 .PHONY: bare-execution-tests
 # Run all our e2e tests in "bare" mode and check for non-zero exit codes.
 bare-execution-tests:
+	PYTHONPATH=. \
```

Comment:
Ah this just caught my eye: this change seems to be unrelated to the PR 🤔

Reply:
I had to include this in order to get `make bare-execution-tests` to pass. It was failing on the newly added import statements in `e2e_playwright/conftest.py`. Adding this fixes it. The other python test commands have this, so this change brings it in line with them. Let me know if you think is problematic though!

Reply:
Ahh ok, thanks for providing the context 🙂